var/spack/repos/builtin/packages/rocm-openmp-extras/package.py

```diff
@@ -224,11 +266,17 @@ def patch(self):
                 aomp_extras.format(src) + '/libm/CMakeLists.txt')
 
         # Openmp adjustments
-        if self.spec.version >= Version('4.5.0'):
+        if (self.spec.version >= Version('4.5.0') and
+            self.spec.version < Version('master')):
             filter_file(
                 '{ROCM_DIR}/amdgcn/bitcode', '{DEVICE_LIBS_DIR}',
                 libomptarget.format(src) + '/deviceRTLs/libm/CMakeLists.txt')
 
+        if self.spec.version >= Version('master'):
```

Comment:
I don't think this is going to be a good idea, since `master` is a moving target